src/components/overview/sub-Components/AddToBag.jsx

```diff
@@ -24,30 +24,45 @@ let orderNumbers = [
   20
 ];
 
+const containerStyle = {
```

Comment:
Do you see any benefit to making a separate file for AddToBag css? If this grows longer and more complex you may have issues with understanding the code.